.github/workflows/code-coverage-windows.yml

```diff
@@ -1,4 +1,4 @@
-name: CI Pipeline Code Coverage Windows
+name: Coverage Windows
```

Comment:
Suggestion to name this "Code Coverage Windows" to make it clear what _kind_ of coverage will be tested.

Reply:
There is nothing else to cover..... I needed smaller names for the badges and this explains perfectly well what this action does